Ports/SDL2/patches/0001-Add-SerenityOS-platform-support.patch

```diff
@@ -141,7 +150,7 @@ index 815665364785ca6e6eb679ed345ff5581ffe235a..9c52ab96fc0694c7400092ef5c513d56
 +
 +    set(CMAKE_CXX_FLAGS "-std=c++23 -fno-exceptions ${CMAKE_CXX_FLAGS}")
 +    file(GLOB SERENITY_SOURCES ${SDL2_SOURCE_DIR}/src/video/serenity/*.cpp ${SDL2_SOURCE_DIR}/src/audio/serenity/*.cpp)
-+    set(SOURCE_FILES ${SOURCE_FILES} ${SERENITY_SOURCES})
++    list(APPEND SOURCE_FILES ${SERENITY_SOURCES})
```

Comment:
Functionally equivalent, using the same mechanism as the surrounding blocks now.